packages/webapp/pages/register.tsx

```diff
@@ -12,13 +12,24 @@ import {
   ProfileHeading,
 } from '@dailydotdev/shared/src/components/utilities';
 import AnalyticsContext from '@dailydotdev/shared/src/contexts/AnalyticsContext';
+import FeaturesContext from '@dailydotdev/shared/src/contexts/FeaturesContext';
+import {
+  Features,
+  getFeatureValue,
+  isFeaturedEnabled,
+} from '@dailydotdev/shared/src/lib/featureManagement';
 import MainLayout from '../components/layouts/MainLayout';
 
 export default function Register(): ReactElement {
   const { user, logout } = useContext(AuthContext);
   const router = useRouter();
   const [disableSubmit, setDisableSubmit] = useState<boolean>(true);
   const { trackEvent } = useContext(AnalyticsContext);
+  const { flags } = useContext(FeaturesContext);
+
+  const getSignupModalFeatureValue = (featureFlag: Features) => {
+    return getFeatureValue(featureFlag, flags);
+  };
```

Comment:
Not sure if we need this function actually...

Reply:
@idoshamun I added that function for readability purpose. But you are right, it may be redundant. Happy to remove it if you feel it is better without :thumbsup:

Reply:
Let it be for now 😁